packages/core/RELEASES.md

```diff
@@ -1,7 +1,8 @@
 # @actions/core Releases
 
-### Unreleased
+### 1.2.0
 
+- Don't throw on set-secret [#141](https://github.com/actions/toolkit/issues/141)
```

Comment:
I wouldn't phrase as "Don't throw".  The release notes should say that set-secret is now available for use.